ifaddr/_posix.py

```diff
@@ -36,9 +38,9 @@ class ifaddrs(ctypes.Structure):
                     ('ifa_addr', ctypes.POINTER(shared.sockaddr)),
                     ('ifa_netmask', ctypes.POINTER(shared.sockaddr))]
 
-libc = ctypes.CDLL(ctypes.util.find_library("socket" if os.uname()[0] == "SunOS" else "c"), use_errno=True)
+libc = ctypes.CDLL(ctypes.util.find_library("socket" if os.uname()[0] == "SunOS" else "c"), use_errno=True)  # type: ignore
 
-def get_adapters(include_unconfigured=False):
+def get_adapters(include_unconfigured: bool = False) -> 'collections._OrderedDictValuesView':
```

Comment:
I think I'll change this to just `Iterable[...]` but I'll merge as is for now.

Reply:
That's actually not a bad idea. In that case it makes probably sense to change it for the Windows version as well, so both have the same type signature (since `Mypy` is currently unhappy about that).